pages/index.js

```diff
@@ -11,4 +12,4 @@ function Home() {
   );
 }
 
-export default withReduxSaga(Home, { type: 'GET_REPO_SERVER' });
+export default withReduxSaga(configureStore, Home, { type: 'GET_REPO' });
```

Comment:
It would be awesome to make this closer to redux api, return a function that accepts the component, a lot of people is used to do that.